diff --git a/node_modules/adm-zip/zipEntry.js b/node_modules/adm-zip/zipEntry.js
index e7804b6..605b02a 100644
--- a/node_modules/adm-zip/zipEntry.js
+++ b/node_modules/adm-zip/zipEntry.js
@@ -59,7 +59,8 @@ module.exports = function (/** object */ options, /*Buffer*/ input) {
 
             // check data integrity
             if (descriptor.compressedSize !== _centralHeader.compressedSize || descriptor.size !== _centralHeader.size || descriptor.crc !== _centralHeader.crc) {
-                throw Utils.Errors.DESCRIPTOR_FAULTY();
+                // This seems to be broken, see https://github.com/cthackers/adm-zip/issues/533
+                // throw Utils.Errors.DESCRIPTOR_FAULTY();
             }
             if (Utils.crc32(data) !== descriptor.crc) {
                 return false;
